mean&Median/mean_median.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"log"
+	"sort"
+	"sync"
+	"time"
+)
+
+func num(v int, wg *sync.WaitGroup, ch1 chan int) {
+	time.Sleep(1 * time.Second)
+	ch1 <- v
+	wg.Done()
+}
+
+func calculation(ch1 chan int, ch2 chan string) {
+	arr := make([]int, 0)
+	var sum, median, n int
+	for v := range ch1 {
+		arr = append(arr, v)
```

Comment:
what is the use of time.Sleep here.

Reply:
So that there will be a 1 second delay after displaying the mean and median value for each input number.